fix compilation with YARP 3.2.101

lrapetti opened the pull request:
Fix for https://github.com/robotology/human-dynamics-estimation/issues/162

lrapetti commented:
After merging in `master` I will merge it also in `devel`

traversaro commented:
Great, thanks. The fix seems to be simpler then the one for wearables, any clue on why?

lrapetti commented:
> Great, thanks. The fix seems to be simpler then the one for wearables, any clue on why?

No, actually was not simpler (I cleaned the PR after we looked at it together). In both I have removed the scoping for the msgs by adding the `CMakeLists.txt` directly in the folder with the `.thrift`. 
Then the changes in the include directories are the following:
- wearables
  I have add the following
  ```
  target_include_directories(WearableData PUBLIC
    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>)
  ```
- HumanDynamicsEstimator
  I have add the following
  ```
  target_include_directories(HumanStateMsg PUBLIC
    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/human>)
    target_include_directories(HumanStateMsg PUBLIC
    $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>)
  ```

For me it is still a bit unclear what is going on.
